spec/features/user_can_view_admin_dashboard_spec.rb

```diff
@@ -1,7 +1,6 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.